quantum/quantum.c

```diff
@@ -209,6 +212,10 @@ bool process_record_quantum(keyrecord_t *record) {
   keypos_t key = record->event.key;
   uint16_t keycode;
 
+  #ifdef VELOCIKEY_ENABLE
+    if (velocikey_enabled()) { velocikey_accelerate(); }
```

Comment:
This should probably only happen on press, or release, not both. 

```suggestion
    if (velocikey_enabled() && record->event.pressed) { velocikey_accelerate(); }
```

Reply:
Also, I'd like to see this moved below the keycode section.  So that it's either right before or right after the preprocess_tap_dance function.